.travis.yml

```diff
@@ -17,17 +18,20 @@ addons:
       - pkg-config
       - gdb
       - python3-pip
+      - python2.7-dbg
+      - python3-all-dbg
       - pylint
       - git
       - cmake
       - libglib2.0-dev
 
 before_script:
-  - pip3 install --user --upgrade ropper retdec-python keystone-engine capstone unicorn
+  #- pip3 install --user --upgrade ropper retdec-python keystone-engine capstone unicorn
+  - /usr/bin/python3 -m pip install --user capstone keystone-engine ropper unicorn
   - echo "source `pwd`/gef.py" > ~/.gdbinit
 
 script:
-  - gdb -q -ex 'gef help' -ex 'gef config' -ex start -ex continue -ex quit /bin/ls
+  - gdb -q -ex 'gef missing' -ex 'gef help' -ex 'gef config' -ex start -ex continue -ex quit /bin/ls
```

Comment:
Do we need this? other than for your test.

Reply:
Let’s agree to keep __gef missing__ in place until we have a config that consistently has no commands missing.  If we ca agree to keep __ropper__ in then we can take __gef missing__ out.